client/api/request.js

```diff
@@ -51,9 +52,8 @@ const _request = ( url, data, method, namespace = '' ) => {
 
 			if ( 403 === response.status ) {
 				const message = __(
-					'The request was not fulfilled. ' +
-					'Please see the status page for diagnostics: ' +
-					'WooCommerce » Status » WooCommerce Services.'
+					'The request was not fulfilled. Please see the {{a}}status page{{/a}} for diagnostics.',
+					{ components: { a: <a href="?page=wc-status&tab=connect" /> } }
```

Comment:
Do we not need to specify `admin.php` here?

Ref: https://github.com/Automattic/woocommerce-services/blob/04e81b836abe0fd01c5e707be5a251dbcacd37d3/client/apps/plugin-status/health.js#L21

Reply:
Fixed in https://github.com/Automattic/woocommerce-services/pull/1369/commits/800aed16a7dfee1d89d23317d339a000ced1ff54